src/components/split-button/split-button-test.stories.tsx

```diff
@@ -53,44 +54,39 @@ export default {
   },
 };
 
-/* eslint-disable react/prop-types */
+type SplitButtonStoryArgs = {
+  buttonType?: SplitButtonProps["buttonType"];
+  textContent: string;
+  text: string;
+  subtext: string;
+};
+
 export const SplitButtonStory = ({
   buttonType,
-  dataElement,
-  dataRole,
   text,
   subtext,
-  iconType,
-  iconPosition,
   ...args
-  /* eslint-enable */
-}) => {
-  return (
-    <Box height={400} mt={100} ml={100}>
-      <SplitButton
-        buttonType={buttonType}
-        data-element={dataElement}
-        data-role={dataRole}
-        text={text}
-        subtext={subtext}
-        iconType={iconType}
-        iconPosition={iconPosition}
-        {...args}
-        onClick={action("click")}
-      >
-        <Button {...args} onClick={action("click")}>
-          Example Button
-        </Button>
-        <Button {...args} onClick={action("click")}>
-          Example Button with long text
-        </Button>
-        <Button {...args} onClick={action("click")}>
-          Short
-        </Button>
-      </SplitButton>
-    </Box>
-  );
-};
+}: SplitButtonStoryArgs) => (
```

Comment:
why we just can't use `Partial<SplitButtonProps>`?

Reply:
Changed :)